### PR TITLE
Removed unnecessary files that aren't python3 compatible

### PR DESCRIPTION
**Pull request by sgill2**
In order to make blues conda installable (specifically for python 3) this PR removes unnecessary files that aren't python3 compatible.

### CODE REVIEW

**Comment by sgill2** [comment]
Yeah I'm not too sure either. I'm rerunning the tests for an unrelated open PR to see if the issues persist; then maybe it's a problem with git/travis instead.

**Comment by davidlmobley** [comment]
I'm testing something, will close and re-open.

**Comment by davidlmobley** [comment]
I just switched to using travis-ci.org rather than travis-ci.com (the latter being for "companies" -- basically for private repos; I was using it while this was still a private repo). It's possible I'd run out of builds and/or time on the travis-ci.com trial I was using, but since that's no longer necessary I've moved over. Unsure if it will help, but we'll see.

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def find_package_data(data_root, package_root):
     license = "GNU Lesser General Public License (LGPL), Version 3",
```
Comment (davidlmobley) [inline setup.py]: This is not GPL; I think we have it under MIT. Please fix.

**Pull request merged**